src/snowflake/snowpark/modin/plugin/_internal/utils.py

```diff
@@ -284,8 +285,9 @@ def _create_read_only_table(
             else STATEMENT_PARAMS.UNKNOWN,
         }
         statement_params.update(new_params)
+        # TODO: use snowpark save_as_table when user facing scoped object is supported
```

Comment:
Can we have a jira for this TODO?

Reply:
+1

Reply:
actually, that is not a valid TODO at this moment. SCOPE is really implemented as internal feature, and have a very restricted use pattern. save_as_table is a public user facing interface, which may not be something could be exposed at this moment. 
Removed this todo for now